components/omega/doc/design/VerticalMixingCoeff.md

```diff
@@ -0,0 +1,182 @@
+
+With the definition of the gradient richardson number, the viscosity and diffusivity are defined according to [Pacanowski and Philander (1981)](https://journals.ametsoc.org/view/journals/phoc/11/11/1520-0485_1981_011_1443_povmin_2_0_co_2.xml?tab_body=pdf) as
+
+$$
+\nu = \frac{\nu_o}{(1+\alpha Ri)^n} + \nu_b
+$$
+
+$$
+\kappa = \frac{\nu}{(1+\alpha Ri)} + \kappa_b
+$$
+
+in these formulae, $\alpha$ and *n* are tunable parameters, most often chosen as 5 and 2 respectively. $\nu_b$ and $\kappa_b$ are the background viscosity and diffusivity respectively. MPAS-Ocean has a $\kappa_{b,passive}$ for applying to just the passive tracers, leaving the active tracers with $\kappa_b$.
+
+### 3.2 Convective Instability Mixing
+
+Commonly, mixing due to convective instability is treated as a step function for the diffusivity and viscosity. This is often represented as
+
+$$
+\kappa =
+\begin{cases}
+\kappa_{conv} \quad \text{ if } N^2 \leq N^2_{crit}\\
+0 \quad \text{ if } N^2 > N^2_{crit}
+\end{cases}
+$$
+
+A similar expression is utilized for viscosity. The effect of this formula is to homogenize T, S, passive tracers, and normal velocity, for unstable stratification, but also in neutral stratification. The behavior in unstable stratification is likely correct, but for neutral stratification, homogenization of momentum is questionable. To allow for a different behavior, we slightly modifty the algorithm above to,
+
+$$
+\kappa =
+\begin{cases}
+\kappa_{conv} \quad \text{ if } N^2 < N^2_{crit}\\
+0 \quad \text{ if } N^2 \geq N^2_{crit}
+\end{cases}
+$$
+
+$N^2_{crit}$ is typically chosen to be 0.
+
+## 4 Design
+
+Vertical chunking, as is, does not work well for vertical derivatives, so a first design of the vertical mixing coefficients computation does not do vertical chunking and just computes with the full-depth column. 
+
+Additionally, to start, only the down gradient contribution to vertical mixing will be added, with contributions to the vertical viscosity and diffusivity coming from the shear (Richardson number mixing), convective, and background mixing models detailed in the prior section. However, in future developments, the K Profile Parameterization [(KPP; Large et al., 1994)](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/94rg01872) and other non-local and/or higher-order mixing models will be added. Some of these parameterizations require vertical derivatives, full-depth integrals, and/or formulate $\kappa$ and $\gamma$ in Eq. (1) Section 2.1 as functions of surface forcing, which require either full-depth column or surface forcing information at depth, thus we design the mixing coefficients routine with this in mind. A solution that allows chunking with vertical operations such as derivatives and integrals and/or surface forcing values would be advantageous, but is outside of the scope of this design document.
+
+### 4.1 Data types and parameters 
+
+#### 4.1.1 Parameters
+
```

Comment:
Will all these parameters be run-time configurable through the yaml config?

Reply:
Yes, they are run-time configurable through the yaml config, this and a specification that a new yaml section should be added are now on line 109.